server/services/utils/getServicesForUser.ts

```diff
@@ -247,9 +247,9 @@ export default (
     },
     {
       id: 'submit-an-intelligence-report',
-      heading: 'Submit an Intelligence Report',
-      description: 'Access to the new Mercury submission form',
-      href: config.serviceUrls.mercurySubmit.url,
+      heading: 'Intelligence Management Service',
+      description: 'Manage and view intelligence reports',
+      href: config.serviceUrls.imsSubmit.url,
       enabled: () => true,
```

Comment:
So this link needs only be shown for users with the IMS User Role. See Pathfinder config for example

Reply:
Also, just spotted that you have edited the submit Mercury config.. that's not right. You need new config definition for the IMS service link